server/src/hardware/protocol/z21/messages.hpp

```diff
@@ -1423,6 +1444,69 @@ constexpr std::string_view toString(LanXSetLocoFunction::SwitchType value)
   return {};
 }
 
+struct MessageReplyType
+{
+  enum Priority
+  {
+    LowPriority = 0,
+    Normal = 1,
+    Urgent = 2
+  };
+
+  enum Flags
+  {
+    CheckDb0       = 1,
+    CheckSpeedStep = 1 << 1,
+    CheckAddress   = 1 << 2
+  };
+
+  inline Priority getPriority() const
+  {
+    return Priority(m_flags & 0xF);
+  }
+
+  inline void setPriority(Priority priority)
```

Comment:
as the above is change, the argument name causes a *shadow* warning, that's why the common practice in the Traintastic codebase is to use `value` as name for the setter argument.